users/kimurayoshimi/12.js

```diff
@@ -1,15 +1,22 @@
 function f(input) {
-  const result = input.reduce((target, key) => {
-    target[key[0]] = key[1];
+  const result = input.reduce((target, [key, value]) => {
+    target[key] = value;
```

Comment:
I used Tuple based on the reviews.

## [stackblitz](https://stackblitz.com/edit/node-qcr24v?file=users%2Fkimurayoshimi%2F12.js)

## Run-time commands
```
https://stackblitz.com/edit/node-qcr24v?file=users%2Fkimurayoshimi%2F12.js
```